src/sections/Hero.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useEffect, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import useWindowDimensions from "~/hooks/useWindowDimensiosn";
+
+const Hero = () => {
+  const { width } = useWindowDimensions();
+
+  const [fireClass, setFireClass] = useState({
+    display: "none",
+    left: "",
+    top: "",
+    translate: "",
+    scale: "",
+  });
+  const [tree1Class, setTree1Class] = useState({
+    display: "none",
+    left: "",
+    top: "",
+    translate: "",
+    scale: "",
+  });
+  const [tree2Class, setTree2Class] = useState({
+    display: "none",
+    right: "",
+    top: "",
+    translate: "",
+    scale: "",
+  });
+
+  const setFirePos = (width: number) => {
+    let firePosX, firePosY;
+    let fireScale = width / 1440;
+    if (width >= 768) {
+      // desktop
+      firePosX = (915 / 1440) * width;
+      firePosY = (600 / 1617) * ((1617 / 1440) * width);
+    } else {
+      // mobile
+      firePosX = (168 / 375) * width;
+      firePosY = (375 / 821) * ((821 / 375) * width);
+      fireScale = (width / 375) * 0.5;
+    }
+
+    setFireClass({
+      display: "block",
+      left: `${firePosX}px`,
+      top: `${firePosY}px`,
+      translate: "-50% -50%",
+      scale: `${fireScale}`,
+    });
+  };
+
+  const setTree1Pos = (width: number) => {
+    let tree1PosX, tree1PosY;
+    let tree1Scale = width / 1440;
+
+    if (width >= 768) {
+      // desktop
+      tree1PosX = (0 / 1440) * width;
+      tree1PosY = (1050 / 1617) * ((1617 / 1440) * width);
+    } else {
+      // mobile
+      tree1PosX = (12 / 375) * width;
+      tree1PosY = (650 / 821) * ((821 / 375) * width);
+      tree1Scale = width / 375;
+    }
+
+    setTree1Class({
+      display: "block",
+      left: `${tree1PosX}px`,
+      top: `${tree1PosY}px`,
+      translate: "-50% -50%",
+      scale: `${tree1Scale}`,
+    });
+  };
+
+  const setTree2Pos = (width: number) => {
+    let tree2PosX, tree2PosY;
+    let tree2Scale = width / 1440;
+
+    if (width >= 768) {
+      // desktop
+      tree2PosX = (0 / 1440) * width;
+      tree2PosY = (1000 / 1617) * ((1617 / 1440) * width);
+    } else {
+      // mobile
+      tree2PosX = (0 / 375) * width;
+      tree2PosY = (560 / 821) * ((821 / 375) * width);
+      tree2Scale = width / 375;
+    }
+
+    setTree2Class({
+      display: "block",
+      right: `${tree2PosX}px`,
+      top: `${tree2PosY}px`,
+      translate: "50% -50%",
+      scale: `${tree2Scale}`,
+    });
+  };
+
+  useEffect(() => {
+    if (width) {
+      setFirePos(width);
+      setTree1Pos(width);
+      setTree2Pos(width);
+    }
+  }, [width]);
+
+  return (
+    <>
+      <div className="relative -z-10 w-full overflow-x-clip">
+        <Image
+          priority
```

Comment:
The SVG rendering performance is subpar due to complexity, it might be better to rasterize into webp/jpegxl. @err53 what do you think? You're great with performance stuff